SPM Example/PortalSwift/ViewController.swift

```diff
@@ -2032,29 +2167,13 @@ class ViewController: UIViewController, UITextFieldDelegate {
           return
         }
 
-        let transactionParam = BuildTransactionParam(
+        let transactionHash = try await portal.sendSol(
```

Comment:
why is this going back to usign sendSol, is it because we are using build transaction under the hood?